todoman/configuration.py

```diff
@@ -83,4 +83,18 @@ def load_config():
                 'Bad {} setting, {}'.format(key, error.args[0])
             )
 
+    date_format = config['main']['date_format']
```

Comment:
I'd rather see new validators that include this in this block (eg: like `validate_colour` and `validate_cache_path`.
Mostly for separation of concerns, and to avoid this function growing too much.